backend-models/service-tests/fileedservice-tests.txt

```diff
@@ -121,14 +121,41 @@ if ($result && $_SESSION['uid'] == 1) {
 Test the user's access to a course (line 39)
 --------------------------------------------------------------------------------------------------------------------------------------------
 
+*********
+* pre-req:
+*********
 
+$cid = 5678;
+$userid = 1234;
+$expectedResults = [
+    'w' => false, // should be false if the user does not have 'w' access level
+    'st' => true, // should be true if the user has 'st' access level
+    'sv' => true, // should be true if the user has 'sv' access level
+];
 
-- Test 4: Test the user's access to a course:
-	1. Start up the session.
-	2. Set the user ID in the session.
-	3. Call the function "hasAccess()". Try different access levels.
-	4. Examine the returning values.
-	5. For the test to pass, the returning value should be false if the user does not have access to the course and true if the user has.
+
+*********
+* Start up the session:
+*********
+
+session_start();
```

Comment:
just copying in session_start();  is not helpful

![image](https://user-images.githubusercontent.com/102600391/234231038-4b8860ad-33ef-425c-906a-2adb38855b2a.png)

what value does the user id need to have to be able to execute the service ?
this depending on what privileges are needed (probably something to do with $hasAccess).

if anything this is part of the pre-req